package.json

```diff
@@ -37,7 +37,7 @@
     "localforage-observable": "2.0.1",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#87769262aa02e1784570f61f4f962050e07cc335",
-    "mattermost-redux": "github:mattermost/mattermost-redux#40368bb807b8b5a3dc0f4ce4a5a76eabc5c9de54",
+    "mattermost-redux": "github:mattermost/mattermost-redux#495176305523322d09706c0416d3acd42e56a716",
```

Comment:
You can update this now to the master branch head in the mattermost-redux repo. Other than that, looks good to me. Thanks @dizkek!

Reply:
I changed but there is still some type errors....